UnityProject/Assets/Scripts/Items/SuitSensor.cs

```diff
@@ -184,24 +184,21 @@ private IDCard GetIdentification()
 			return null;
 		}
 
-
-		private void SwitchMode(PlayerScript player)
+		public void SwitchMode(Vector2 mousePosition)
```

Comment:
this is not used ServerPerformInteraction(HandActivate interaction)

Reply:
It's used by the action

Reply:
ok cool